cueweb/.env.example

```diff
@@ -0,0 +1,23 @@
+NEXT_PUBLIC_OPENCUE_ENDPOINT=http://your-rest-gateway-url.com
+
+SENTRY_ENVIRONMENT='development'
+
+
```

Comment:
It's worth adding a comment saying not all options are required and that if none of them are configured, the feature is disabled.

Reply:
Done!

See commit: [cueweb] Make CueWeb authentication and Sentry usage optional
https://github.com/AcademySoftwareFoundation/OpenCue/pull/1596/commits/9c326b6b4e072563281fa3a92ac8cbd8c93f6ef3

- Added functionality to make login and Sentry usage optional in CueWeb.
- Updated configuration to enable/disable these features via environment variables.
- Enhanced the cueweb/README.md with instructions for configuring these options in the CueWeb setup.

### About Sentry usage in CueWeb:

- Sentry environment variables
   - If you use the [Sentry](https://sentry.io/) system to monitor the CueWeb system, the following environment variables should be set:
      - SENTRY_ENVIRONMENT
      - SENTRY_URL
      - SENTRY_DSN
      - SENTRY_ORG
      - SENTRY_PROJECT
   - If you do not want to use Sentry, do not define `SENTRY_DSN`.

Change `.env` 

From:
```env
# Sentry values
SENTRY_ENVIRONMENT='development'
SENTRY_DSN = sentrydsn
SENTRY_URL = sentryurl
SENTRY_ORG = sentryorg
SENTRY_PROJECT = sentryproject
```

To:

```env
# Sentry values
SENTRY_ENVIRONMENT='development'
# SENTRY_DSN = sentrydsn
SENTRY_URL = sentryurl
SENTRY_ORG = sentryorg
SENTRY_PROJECT = sentryproject
```